cachito/workers/pkg_managers/pip.py

```diff
@@ -1448,6 +1477,115 @@ def _sdist_preference(sdist_pkg):
     return yanked_pref, filetype_pref
 
 
+def _download_vcs_package(requirement, pip_deps_dir, pip_raw_repo_name, nexus_auth):
+    """
+    Fetch the source for a Python package from VCS (only git is supported).
+
+    After downloading, upload this package to the Nexus raw repository and on subsequent
```

Comment:
can we also say we will upload the package to the Nexus hoster instance if 2 nexus instances are being used?

Reply:
Yup, done. Side question, what exactly does the term "hoster" mean in this context?

Reply:
That is how we named all the variables that refer to the cachito instance that hosts the permanent Cachito repositories

Reply:
Makes sense, thanks :-)